_layouts/post.liquid

```diff
@@ -18,8 +18,9 @@ layout: default
   <header class="post-header">
     <h1 class="post-title">{{ page.title }}</h1>
     <p class="post-meta">
-      {{ page.date | date: '%B %d, %Y' }}
+      Created: {{ page.date | date: '%B %d, %Y' }}
```

Comment:
I think it would be better as `Created in` instead of `Created:`

Reply:
I have modified it according to your suggestion.